proto/interchain_security/ccv/provider/v1/provider.proto

```diff
@@ -59,6 +59,10 @@ message ConsumerAdditionProposal {
     // This param is a part of the cosmos sdk staking module. In the case of 
     // a ccv enabled consumer chain, the ccv module acts as the staking module.
     int64 historical_entries = 13;
+    string conn_id = 15; // the ID of the connection end on the consumer chain on top of which the CCV channel will be established.
+    string transfer_channel_id = 16; // the ID of a token transfer channel used for the Reward Distribution sub-protocol.
+    // TODO: preCCV field shouldn't be added here? to make sure if it's a new chain with consumer initial validator set
```

Comment:
I believe you're correct in that preCCV shouldn't be added here, CC @mpoke

It seems like that field only concerns the consumer

Reply:
preCCV is not needed in ConsumerAdditionProposal.  Cf. [spec](https://github.com/cosmos/ibc/blob/main/spec/app/ics-028-cross-chain-validation/data_structures.md#ccv-data-structures), 
> `connId` is the ID of the connection end on the provider chain on top of which the CCV channel will be established. If `connId == ""`, a new client of the consumer chain and a new connection on top of this client are created. Note that a sovereign chain can transition to a consumer chain while maintaining existing IBC channels to other chains by providing a valid `connId`.